readme.md

```diff
@@ -754,25 +754,31 @@ To build OSARA, you will need:
 
 - Several git submodules used by OSARA.
 	See the note about submodules in the previous section.
-- Windows only: Build Tools for Microsoft Visual Studio 2022:
-	* [Download Build Tools for Visual Studio 2022](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022)
+
+#### Windows:
+- Build Tools for Microsoft Visual Studio 2022: [Download Build Tools for Visual Studio 2022](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022)
 	* Visual Studio 2022 Community/Professional/Enterprise is also supported.
-	* Whether installing Build Tools or Visual Studio, you must enable the following:
-		- In the list on the Workloads tab, in the Windows grouping: Desktop development with C++
-		- Then in the Installation details tree view, under Desktop development with C++ > Optional:
-			* C++ ATL for latest v143 build tools (x86 & x64)
-			* C++ Clang tools for Windows
-- Mac only: Xcode 13:
-	* You can download Xcode 13 from the [Mac App Store](https://apps.apple.com/us/app/xcode/id497799835?ls=1&mt=12).
+- Whether installing Build Tools or Visual Studio, you must enable the following:
```

Comment:
Note that this list item is related to visual studio, therefore this text should belong to the first bullet point. instead. I wonder whether this can just be a paragraph?

Reply:
```suggestion
	Whether installing Build Tools or Visual Studio, you must enable the following:
```